src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -426,25 +601,40 @@ export class SavedObjectsRepository {
     }
 
     const { refresh = DEFAULT_REFRESH_SETTING } = options;
-
     const allTypes = Object.keys(getRootPropertiesObjects(this._mappings));
```

Comment:
Yes, as far as I can tell these should be equivalent, so we should prefer to use the registry. @pgayvallet Do you know if there was a reason we didn't use the registry, or is it just some refactoring that we still need to do?

Since you're already busy with a huge pull request it might be best to leave this as is, and once you've merged this PR we can refactor this to avoid unnecessary conflicts.

Reply:
AFAIK, since the `config` type has been migrated to a real SO type, we should be good to use the registry instead. This seems to just be somethings I missed. `getRootPropertiesObjects` should no longer be used to list the SO types anywhere.

But I agree with rudolf, this can be done at a later time by the platform team.